parachain/primitives/core/src/lib.rs

```diff
@@ -26,3 +27,7 @@ pub trait Verifier {
 		descendants_until_final: u8,
 	) -> Result<(), &'static str>;
 }
+
+pub trait SubmitMessage {
```

Comment:
this trait name is a bit too simple. For example it doesn't describe where the message is being submitted.

```suggestion
pub trait OutboundQueue {
```

Reply:
addressed in [1a59f22](https://github.com/Snowfork/snowbridge/pull/805/commits/1a59f22c260a53df423de9a590b31b55ab1c9627)